### DISCUSSION TITLE
After accepting a classroom assignment I get a 500 error

### DISCUSSION THREAD

**Discussion by Web-Dev-Codi** [community/community#130738] [GitHub Education] [answered]
### Hi everyone,

After accepting a Github classroom assignment, I am redirected to a 500 internal error page. I checked github status and everything seems operational. What are my next steps? wait? 
![Screenshot from 2024-07-02 12-36-01](https://github.com/community/community/assets/80214475/fe244116-2cd7-44f2-a6eb-3a91477c9895)

**Comment by RyanHecht** [comment, marked as answer]
Hi @Web-Dev-Codi! Are you still receiving a 500 error? Due to a recent change, your instructor may need to enable forking private repositories in the classroom organization. We just changed it so you should get a message if that's the case now!

**Reply by Web-Dev-Codi** [reply]
Everything is working fine and the error was resolved in a short time. Thank you for reaching out and taking time to resolve this issue. Enjoy your day.

**Reply by I0x0I** [reply]
Our students are having this problem starting around 9am (CST) this morning while everything was OK before 9am and last night, could you give us some suggestion on resolving it?

**Reply by sergeichuprov** [reply]
Have the same problem from today's morning. Any ideas how the issue can be resolved?

**Reply by StephenUTSA** [reply]
Having the same problem!

**Reply by BaeztheProg** [reply]
The change to accept private repos did not fix.

**Reply by sfogarty** [reply]
ALso having this problem at Trinity  having this problem, and we were already set to accept private repos.

**Reply by OberonDixon** [reply]
I also just started having this issue with students today, see my comment here: https://github.com/orgs/community/discussions/137628#discussioncomment-10538046

**Comment by Yusofuani** [comment]
hello iam still getting this message anyway  I can fix it?

**Comment by chrissphinx** [comment]
@RyanHecht 
![CleanShot 2024-09-03 at 14 40 41@2x](https://github.com/user-attachments/assets/f8e42a07-8351-4f7c-a7b4-2cf193701198)

Mine is enabled and we're currently experiencing this issue. This is true for both organizations I have for courses. Please fix this soon

**Reply by egrace479** [reply]
Same for my class! We have many students in who have both submitted and not submitted their assignments, while many are now experiencing this 500 error or it's asking them to wait while their repo is set up. 

On my end I see repositories created for my students, but they're not connected to the classroom and the students don't have access. I also tried manually adding one student to his repo (giving him write access). He submitted the assignment and was autograded, but it still won't show up in the classroom.

**Comment by StephenUTSA** [comment]
This issue is happening in a class where I am trying introduce students to git and github oooof

**Reply by pinventado** [reply]
Same here :)  A nice introduction of GitHub to 244 students in my case. Plus all other students who are affected by this bug.

**Comment by sfogarty** [comment]
This is also being discussed in https://github.com/orgs/community/discussions/137628, which seems to be the newer discussion.

**Comment by chrissphinx** [comment]
this bug is back, students are not being added as collaborators to their created repos, even though the repo is created cc: @RyanHecht

**Comment by pinventado** [comment]
Yup, bug is back for us again too. Has anyone written a script to automate giving students access to their repos? Might be useful for large classes like mine.
